pkg/openthread/include/platform_config.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) Baptiste Clenet
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     net
+ * @file
+ * @brief       Implementation of OpenThread platform config
+ *
+ * @author      Baptiste Clenet <bapclenet@gmail.com>
+ * @}
+ */
+
+#ifndef OPENTHREAD_CORE_PLATFORM_CONFIG_RIOT_H_
+#define OPENTHREAD_CORE_PLATFORM_CONFIG_RIOT_H_
+
+/**
+ * @def OPENTHREAD_CONFIG_NUM_MESSAGE_BUFFERS
+ *
+ * The number of message buffers in buffer pool
+ */
+#ifdef OPENTHREAD_ENABLE_NCP_UART
+#define OPENTHREAD_CONFIG_NUM_MESSAGE_BUFFERS       5
+#elif defined(OPENTHREAD_FTD) || defined(OPENTHREAD_MTD)
+#define OPENTHREAD_CONFIG_NUM_MESSAGE_BUFFERS       15
+#endif
+
+/**
+  * @def OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE
+  *
+  * Define to 1 if you want use legacy transmit done.
+  *
+  */
+#define OPENTHREAD_CONFIG_LEGACY_TRANSMIT_DONE 1
+
+#endif /* OPENTHREAD_CORE_PLATFORM_CONFIG_RIOT_H_ */
```

Comment:
trailing underscore. By the way, I think this header guard name won't pass the CI. I should be `PLATFORM_CONFIG_H`